tests/pkg_libcose/main.c

```diff
@@ -40,16 +40,19 @@ static unsigned char pk[COSE_CRYPTO_SIGN_ED25519_PUBLICKEYBYTES];
 static unsigned char sk[COSE_CRYPTO_SIGN_ED25519_SECRETKEYBYTES];
 static unsigned char pk2[COSE_CRYPTO_SIGN_ED25519_PUBLICKEYBYTES];
 static unsigned char sk2[COSE_CRYPTO_SIGN_ED25519_SECRETKEYBYTES];
-static unsigned char symmkey[COSE_CRYPTO_AEAD_CHACHA20POLY1305_KEYBYTES];
-static uint8_t nonce[COSE_CRYPTO_AEAD_CHACHA20POLY1305_NONCEBYTES] = { 0 };
 /* COSE structs */
 static cose_sign_enc_t sign;
 static cose_sign_dec_t verify;
 static cose_signature_t signature1, signature2;
-static cose_key_t signer1, signer2, symm;
+static cose_key_t signer1, signer2;
+#if defined(MODULE_LIBCOSE_CRYPT_HACL) || defined(MODULE_LIBCOSE_CRYPT_MONOCYPHER)
```

Comment:
Can I have a bit of whitespace before the `#if` and after `#endif`? Feel free to squash immediately

Reply:
addressed.